### DISCUSSION TITLE
部分用户加载评论出错

### DISCUSSION THREAD

**Discussion by ShuiLinzi** [walinejs/waline#926] [Q&A] [answered]
我对象访问我博客的时候评论加载不出来，但是其他大部分人都可以，控制台报错如下
https://cdn.jsdelivr.net/gh/ShuiLinzi/blog-image@master/后端/无法加载.webp
求大大解答

**Comment by lizheming** [comment, marked as answer]
你用的是 Valine 吧，这边是 Waline，不负责解答 Valine 相关问题
